app/views/support/locations/new.html.erb

```diff
@@ -1,10 +1,13 @@
 <%= render PageTitle.new(title: "support.providers.locations.new") %>
 
-<h1 class="govuk-heading-l">Add a location</h1>
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l"><%= "#{@provider.provider_name} (#{@provider.provider_code})" %></span>
+    Add location
```

Comment:
![image](https://user-images.githubusercontent.com/470137/226671624-84b1ff6d-d245-4ce1-9fbe-f82096c98de4.png)

Reply:
Double oops, there was no back link on the prototype

Reply:
I think that might be by design, as the back link could go to either the index page or the check answers page, where it clears the location_details